dali/kernels/imgproc/roi.h

```diff
@@ -43,16 +43,46 @@ namespace kernels {
 template <int ndims>
 using Roi = Box<ndims, int>;
 
+template <int n>
+DALI_HOST_DEV
+ivec<n> shape2vec(const TensorShape<n> &shape) {
+  ivec<n> ret;
+  for (int i = 0; i < n; i++)
+    ret[n-1-i] = shape[i];
+  return ret;
+}
+
+template <int n>
+DALI_HOST_DEV
+TensorShape<n> vec2shape(const ivec<n> &shape_vec) {
```

Comment:
shape2vec and vec2shape have only in and out swaped out. Can it be made in one function template?